apps/forwarder-sriov/forwarder.yaml

```
@@ -13,6 +13,7 @@ spec:
     metadata:
       labels:
         app: forwarder-sriov
+        "spiffe.io/spiffe-id": "true"
```

Apply thisalso  for other places.
```suggestion
        spiffe.io/spiffe-id: true
```

---

key `spiffe.io/spiffe-id` is not recognized without quotes

---

ok, lets then update "true" to true

---

ok, done